.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+on:
+  push:
+    branches: [ master ]
+    tags: [ v* ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I know you copied this across, but it's still surprising that it should appear to say "publish the package when a pull request is opened".

Reply:
You want to know that the build will work on the PR, the actual publish step then cancels unless we're on master

Reply:
Is that from `if... startsWith(github.ref, 'refs/tags')`?

Reply:
yeah, that's right. See the skipped part of this build and compare with the build here: https://github.com/mrc-ide/hipercow-py/actions/runs/14133922257/job/39600987440 (on hipercow, built on a release)